VisualStudio.gitignore

```diff
@@ -137,6 +137,9 @@ _TeamCity*
 .axoCover/*
 !.axoCover/settings.json
 
+# Coverlet is a free, cross platform Code Coverage Tool
+coverage*[.json, .xml, .info]
```

Comment:
I'm not 100% sure this is how to define a rule to cover multiple extensions. Here's how I tested it:

```shellsession
$ git init coverlet-test
Initialized empty Git repository in /Users/shiftkey/src/test/coverlet-test/.git/
$ cd coverlet-test
$ echo "coverage*[.json, .xml, .info]" >> .gitignore
$ cat .gitignore
coverage*[.json, .xml, .info]
$ git add .
$ git commit -am "add rule"
[master (root-commit) 5240fe7] add rule
 1 file changed, 1 insertion(+)
 create mode 100644 .gitignore
$ touch coverlet.json
$ git status
On branch master
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	coverlet.json

nothing added to commit but untracked files present (use "git add" to track)
```

I think this needs to be split out to multiple lines to achieve the effect you're interested in.

Reply:
@shiftkey it works fine. you created a file with wrong name, that does not match the rule - you created `coverlet.json`, but the rule is `coverage.json`. Could be typo ;)

![image](https://user-images.githubusercontent.com/5943484/70325856-477d1580-1833-11ea-8da9-50ab1ae5426f.png)

Reply:
> you created a file with wrong name

![](https://media.giphy.com/media/10o3Um2U3wa4DK/giphy.gif)

Well that was silly of me. Serves me right for doing things before I'm properly caffeinated.

Reply:
@lkurzyniec: @shiftkey is correct.  The syntax is wrong.  I discovered this because the pattern is causing source files named "Coverage*.cs" to be ignored.

Reply:
@phoogenb do you think this is something that needs to be revisited? I've not heard any issues about this since it was merged...

Reply:
@shiftkey
unfortunately @phoogenb is right, here is PR to fix that #3415.